adobe/react-spectrum
# fix(#4160): NumberField: screen reader fails to announce negative values with currencySign: 'accounting'

Closes #4160 

## ✅ Pull Request Checklist:


## 📝 Test Instructions:

1. Open https://reactspectrum.blob.core.windows.net/reactspectrum/3e5388ac6912b0f3bff0811f6196dd7bfdb58ad1/storybook/index.html?path=/story/numberfield--currency-switcher&providerSwitcher-express=false
2. Select Currency Sign: Accounting from the dropdown
3. With a screen reader running, decrement the NumberInput until negative accounting values are reached.
4. Screen reader should announce "negative" or "minus" before negative accounting value.


## 🧢 Your Project:

Adobe/Accessibility
